IPython/terminal/embed.py

```diff
@@ -350,7 +350,7 @@ def mainloop(
 def embed(*, header="", compile_flags=None, **kwargs):
     """Call this to embed IPython at the current point in your program.
 
-    The first invocation of this will create an :class:`InteractiveShellEmbed`
+    The first invocation of this will create a :class:`terminal.embed.InteractiveShellEmbed`
```

Comment:
That should be unnecessary, and it seem to work as is with a cross ref, but I have no objections. 

Dit it not work for you ?

Reply:
It did not work for me under `IPython`, only under `IPython.terminal.embed` (after 853291bb3104475994842f3ea9fda44ee938177d, which tweaks the API docs so that `embed()` appears under the `IPython` module in addition to under `IPython.terminal.embed`).